packages/aws-rfdk/lib/deadline/test/repository.test.ts

```diff
@@ -424,7 +424,9 @@ test('repository installer logs all required files', () => {
           {}, // log group name. checked in another test.
           '\",\"log_stream_name\":\"cloud-init-output-{instance_id}\",\"file_path\":\"/var/log/cloud-init-output.log\",\"timezone\":\"Local\"},{\"log_group_name\":\"',
           {}, // log group name again.
-          '\",\"log_stream_name\":\"deadlineRepositoryInstallationLogs-{instance_id}\",\"file_path\":\"/tmp/bitrock_installer.log\",\"timezone\":\"Local\"}]}},\"log_stream_name\":\"DefaultLogStream-{instance_id}\",\"force_flush_interval\":15}}',
+          '\",\"log_stream_name\":\"deadlineRepositoryInstallationLogs-{instance_id}\",\"file_path\":\"/tmp/'+
+          (version.isLessThan(Version.MINIMUM_VERSION_USING_INSTALLBUILDER) ? 'bitrock' : 'installbuilder') +
```

Comment:
I don't think we actually triggering both code paths here, so there's some missed coverage.

I think the value comes from 

https://github.com/aws/aws-rfdk/blob/60a1627376b256a18083fb0914906ec7e87fc2b4/packages/aws-rfdk/lib/deadline/test/repository.test.ts#L121

Reply:
Good catch. I've done a bit of refactoring to support testing with multiple versions and can confirm that the test covers both cases now.